compiler/semtypinst.nim

```diff
@@ -462,6 +462,14 @@ proc propagateFieldFlags(t: PType, n: PNode) =
       propagateFieldFlags(t, son)
   else: discard
 
+template genericParamsToString(t: untyped): untyped =
+  var genericParams = "["
```

Comment:
curious whether there's already a proc for that in compiler/**; i guess u already searched

Reply:
I did search, and I could not find anything unfortunately. 😢  This code was modified from a other code I found that iterated over the generic params.
An argument could be made for moving this helper template into compiler/types.nim which contains the `typeToString()` function.

Reply:
i guess this was resolved by using/extending typeToString